core/src/main/kotlin/org/evomaster/core/Main.kt

```diff
@@ -238,12 +238,34 @@ class Main {
                     info("TCP timeouts: $timeouts")
                 }
 
+                // The number of covered targets and the number of potential faults will be shown in both
+                // blackbox and whitebox
+                val elementsPair = data.find{it -> it.header == "coveredTargets"}
+
+                if (elementsPair != null) {
+                    info("Covered targets (lines, branches, faults, etc.): ${elementsPair.element}")
```

Comment:
that is not applicable to BB. BB has no access to info on `lines, branches`

Reply:
Thank you so much Andrea for your comment, I compared outputs for black-box and white-box. I realized that white-box outputs "coveredTargets" to the console. For the black-box, I saw that information in the Statistics file. So, I thought the issue was not just "potentialFaults", it is all information provided in white-box but not in black-box. Based on that, I just made a fix for displaying "potentialFaults" for the black-box. So, those lines were also removed.